download/downloadSpec.go

```diff
@@ -126,7 +126,14 @@ func detectLinuxId() (string, error) {
 		return "", nil
 	}
 
-	osRelease, osReleaseErr := getOsReleaseContent()
+	osreleaseFile, err := afs.Open("/etc/os-release")
```

Comment:
I'd be tempted to put `"/etc/os-release"` in constant and use that

Reply:
Thanks, I've made it a constant. I don't think it is worth having config for it as it is a well known file and its location won't change (https://www.freedesktop.org/software/systemd/man/os-release.html)